docs/FAQ.md

```diff
@@ -1,5 +1,31 @@
 # FAQ
 
+
+<details>
+<summary>
+What does the plugin do and what user problems does it solve?
+</summary>
+
+The plugin **facilitates the process of transferring shipping information** by providing a direct link between the WooCommerce store and Gebrüder Weiss. This allows the user to **save time and effort**. Furthermore, the plugin provides predefined states for processing and shippment of orders, which are **additional security measures** to ensure that the shipping data will be processed correctly. 
```

Comment:
additional security measures are not provided by the plugin. you could write: 
it makes life for the shop owner easier by automating the process sending gebrüder weiss the order information directly from the woocommerce shop.